app/components/gh-post-settings-menu.js

```diff
@@ -87,13 +87,7 @@ export default Component.extend(SettingsMenuMixin, {
     seoTitle: computed('model.titleScratch', 'metaTitleScratch', function () {
         let metaTitle = this.get('metaTitleScratch') || '';
 
-        metaTitle = metaTitle.length > 0 ? metaTitle : this.get('model.titleScratch');
-
-        if (metaTitle.length > 70) {
-            metaTitle = metaTitle.substring(0, 70).trim();
-            metaTitle = Handlebars.Utils.escapeExpression(metaTitle);
-            metaTitle = htmlSafe(`${metaTitle}&hellip;`);
-        }
+        metaTitle = metaTitle ? metaTitle : this.get('model.titleScratch');
```

Comment:
This could also be written without the ternary by changing the top variable declaration:

 `let metaTitle = this.get('metaTitleScratch') || this.get('model.titleScratch') || '';`